release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -70,6 +70,18 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
 [id="ocp-4-15-web-console"]
 === Web console
 
+[id="ocp-4-14-administrator-perspective"]
+==== Administrator Perspective
+
+With this release, there are several updates to the *Administrator* perspective of the web console. You can:
+
+* Enable and disable the tailing to Pod log viewer to minimize load time.
+* View recomended values for `VerticalPodAutoscaler` on the *Deployment* page.
+
+[id="node-uptime-information"]
+===== Node uptime information
+With this update, you can enable the ability to view additional information related to a nodes uptime on the *Compute* -> *Nodes* page to track node restarts or failures by clicking *Manage columns* and select *Uptime*.
+
```

Comment:
Should I add the Developer hub note with the admin release notes or the dev perspective release notes? As I am looking at it, I think I might should include it with the admin notes. WDYT?